src/application/tally/widgets/proposal.tsx

```diff
@@ -0,0 +1,136 @@
+import { classes, Chip, IconButton, WidgetBase } from 'shared/ui';
+import { getDifferenceInDays, getEndsInLabel } from 'shared/utils';
+
+import { ProposalData } from '../types';
+import {
+  getProposalUrl,
+  getStatusBadgeColorClassNames,
+  getOrganizationUrl,
+  getOrganizationDelegateUrl,
+} from '../utils';
+
+interface Properties {
+  proposalDetails: ProposalData;
+  className?: string;
+  top?: number;
+  isPreviousProposalAvailable: boolean;
+  isNextProposalAvailable: boolean;
+  isLoading: boolean;
+  onClose?: () => void;
+  onPrevious: () => void;
+  onNext: () => void;
+}
+
+export const Proposal = ({
+  proposalDetails,
+  className,
+  top,
+  isPreviousProposalAvailable,
+  isNextProposalAvailable,
+  isLoading,
+  onClose,
+  onPrevious,
+  onNext,
+}: Properties) => {
+  const proposalEndDateInMs = new Date(proposalDetails.end.timestamp).getTime();
+
+  return (
+    <WidgetBase
+      className={classes(
+        'grid h-[200px] overflow-hidden rounded-[0.375rem] bg-white text-xs leading-tight text-tally-text-primary',
+        className,
+      )}
+      top={top}
+      onClose={onClose}
+    >
+      <header className="mb-auto flex items-center justify-between space-x-3">
+        <p className="line-clamp-[1] break-all text-xs text-tally-gray-600">
+          By{' '}
+          <a
+            href={
+              proposalDetails.creator.ens.length > 0 ||
+              proposalDetails.creator.address.length > 0
+                ? getOrganizationDelegateUrl(
+                    proposalDetails.organization.slug ?? '',
+                    proposalDetails.creator.ens.length > 0
+                      ? proposalDetails.creator.ens
+                      : proposalDetails.creator.address,
+                  )
+                : getOrganizationUrl(proposalDetails.organization.slug ?? '')
+            }
```

Comment:
maybe we can hide this conditional logic behind a function `getAuthorUrl` ? This function would receive whole `proposalDetails` obj as argument and return url